Documentation/network/kubernetes/compatibility.rst

```diff
@@ -18,12 +18,30 @@ with Cilium. Older Kubernetes versions not listed in this table do not have
 Cilium support. Newer Kubernetes versions, while not listed, will depend on the
 backward compatibility offered by Kubernetes.
```

Comment:
maybe something like:

Older and newer Kubernetes versions, while not listed, will depend on the
forward / backward compatibility offered by Kubernetes.

Reply:
LGTM